### PR TITLE
readRows implemented

### PR DESCRIPTION
**Pull request by piece-of-tart**

### CODE REVIEW

File: table/src/main/java/tech/ydb/table/Session.java [hunk 1]
```diff
@@ -72,6 +75,12 @@ CompletableFuture<Result<DataQueryResult>> executeDataQuery(
         Params params,
         ExecuteDataQuerySettings settings);
 
+    CompletableFuture<Result<ResultSetReader>> readRows(String pathToTable, List<StructValue> keys,
```
Comment (pnv1) [inline table/src/main/java/tech/ydb/table/Session.java:78]: Its not quite obvious what "StructValue" here should contain. Better provide some comments here.
Could be taken from here: https://github.com/ydb-platform/ydb-api-protos/blob/master/protos/ydb_table.proto#L1056

**Review by pnv1 — CHANGES_REQUESTED** [review]

File: table/src/main/java/tech/ydb/table/Session.java (hunk 1 above)
Comment (piece-of-tart) [reply table/src/main/java/tech/ydb/table/Session.java]: Thanks, added comment

File: table/src/main/java/tech/ydb/table/Session.java [hunk 2]
```diff
@@ -72,6 +74,12 @@ CompletableFuture<Result<DataQueryResult>> executeDataQuery(
         Params params,
         ExecuteDataQuerySettings settings);
 
+/**
+ *      Keys must be a list of structs where each struct is a primary key
```
Comment (pnv1) [inline table/src/main/java/tech/ydb/table/Session.java:78]: No more keys here :)
Comment should be moved to ReadRowsSettings

Comment (piece-of-tart) [reply table/src/main/java/tech/ydb/table/Session.java]: Thanks

**Comment by piece-of-tart** [comment]
Wait readRows in new ydb release

**Review by alex268 — APPROVED** [review]

**Review by pnv1 — APPROVED** [review]

**Pull request merged**